calico-cloud/multicluster/kubeconfig.mdx

```diff
@@ -219,46 +229,137 @@ We'll now create the RemoteClusterConfigurations that establish synchronization
+       kind: Secret
+     syncOptions:
+       overlayRoutingMode: Disabled
+   EOF
+   ```
+
+1. If you have no IP pools in cluster A with NAT-outgoing enabled, skip this step.
+
+   Otherwise, if you have IP pools in cluster A with NAT-outgoing enabled and workloads in that pool will egress to workloads in cluster B, you need to instruct {{prodname}} to not perform NAT on traffic destined for IP pools in cluster B.
+
+   You can achieve this by creating a disabled IP pool in cluster A for each CIDR in cluster B. This IP pool should have NAT-outgoing disabled. For example:
+
+    ```yaml
+    apiVersion: projectcalico.org/v3
+    kind: IPPool
+    metadata:
+      name: clusterB-main-pool
+    spec:
+      cidr: <Cluster B CIDR>
+      disabled: true
+    ```
+
+1. [Validate](#check-remote-cluster-connection) the that the remote cluster connection can be established.
+
+1. Repeat the above steps, switching cluster A and cluster B.
+
+
+After completing the above steps for all cluster pairs in the cluster mesh, your clusters should now be ready to utilize remote-identity-aware policy and federated services.
+
+</TabItem>
+</Tabs>
+
+:::caution
  This tutorial sets up RemoteClusterConfigurations in both directions. This is required for {{prodname}} to manage multi-cluster networking, and also ensures you can write identity-aware policy on both sides of a cross-cluster connection. Unidirectional connections can be made at your own discretion.
 :::
 
-### Validate federation and multi-cluster networking
-#### RemoteClusterConfiguration & federated endpoint identity
-calicoq can be used to ensure that a local cluster has connected to it's configured remote clusters. Use the following command:
+### Switch to multi-cluster networking
+The steps above assume that you are configuring federated endpoint identity and multi-cluster networking for the first time. If you already have federated endpoint identity and are looking to utilize multi-cluster networking, the steps below outline the changes that must be made if you had previously set up federated endpoint identity prior to the introduction of multi-cluster networking:
+
+1. Validate that all [requirements](#calico-enterprise-multi-cluster-networking) for multi-cluster networking have been met.
+2. Update the ClusterRole in each cluster in the cluster mesh using the RBAC manifest found in [Create kubeconfig files](#create-kubeconfig-files)
+3. Update all RemoteClusterConfigurations to set `Spec.OverlayRoutingMode` to `Enabled`
+4. Ensure that all RemoteClusterConfigurations are bidirectional, i.e. ensure that RemoteClusterConfigurations were established following the [instructions](#create-remoteclusterconfigurations) for in both directions for each cluster pair.
+5. If you had previously created disabled IP pools to prevent NAT outgoing from applying to remote cluster destinations, those disabled IP pools are no longer needed when using multi-cluster networking and must be deleted.
```

Comment:
```suggestion
5. If you have previously created disabled IP pools to prevent NAT outgoing from applying to remote cluster destinations, you must delete the disabled IP pools (as they are no longer needed).
```